sig-community/charter.md

```diff
@@ -0,0 +1,60 @@
+# SIG Community
+
+## Scope
+
+This SIG cares about all aspects of what is helpful and harmful to this open source software community.
+We understand that this project is a human endeavor, and it is our job to put people before practice, process, and even principle.
```

Comment:
What does it mean to "put people before principle"? If a principle is a fundamental belief held by the org, then isn't it a principle of the org to put people before practice and process? But, maybe I'm misunderstanding the meaning of "principle" here though.    

Maybe "and it is our job to put people before practice, process, and even progress". is an alternative?

Reply:
This is a nuanced reading, thank you. I didn't think of that recursive loop that might even cancel itself out, in a rules engine. I like your alternative, as it gets to the more radical community reliance and supportiveness I had in mind:

```suggestion
We understand that this project is a human endeavor, and it is our job to put people before practice, process, and even progress.
```

I do think there is a discussion to be had about if the principle of putting people first simple overrides other principles (like [Azimov's Three Laws of Robotics](https://en.wikipedia.org/wiki/Three_Laws_of_Robotics)?), but perhaps that's too nuanced for here and now? Or splitting too many hairs? It would like like this:

```suggestion
We understand that this project is a human endeavor, and it is our primary principle to put people before practice, process, and even other principles.
```

Reply:
Going with first suggestion, as per SIG Community meeting, philosophy for the future.